### DISCUSSION TITLE
FAILED to reload uwsgi

### DISCUSSION THREAD

**Discussion by SapeNeCo** [openwisp/ansible-openwisp2#484] [Q&A] [answered]
Hello.
I'm diving further into ansible openwisp2 and encountered this error when building:
```cmd
RUNNING HANDLER [openwisp.openwisp2 : Reload uwsgi] ********************************************************************************************************************************************************
fatal: [192.168.0.92]: FAILED! => {"changed": true, "cmd": ["/opt/openwisp2/env/bin/uwsgi", "--reload", "/opt/openwisp2/uwsgi.pid"], "delta": "0:00:00.013943", "end": "2024-07-23 08:56:49.614406", "msg": "non-zero return code", "rc": 1, "start": "2024-07-23 08:56:49.600463", "stderr": "open(\"/opt/openwisp2/uwsgi.pid\"): No such file or directory [core/io.c line 525]", "stderr_lines": ["open(\"/opt/openwisp2/uwsgi.pid\"): No such file or directory [core/io.c line 525]"], "stdout": "", "stdout_lines": []}
```
He says that he cannot find the uwsgi.pid file and it really does not exist, why is it not created?

**Comment by SapeNeCo** [comment, marked as answer]
Okay, the problem was in two services:
1. supervisor
2. nginx

After adding modules (in my case it was openwisp-wifi-login-pages), they create configs for them and they must be removed manually if you decide to remove these modules.
They are in
1. /etc/supervisor/conf.d/
2. /etc/nginx/sites-enabled/

Don't forget to delete them if you have disabled the module.
I hope this helps someone.

**Reply by nemesifier** [reply]
Thanks for updating this.
